packages/design-system/src/components/primitives/numeric-gesture-control.ts

```diff
@@ -127,17 +157,15 @@ export const numericScrubControl = (
 
         onStatusChange?.("scrubbing");
         targetNode.addEventListener("pointermove", handleEvent);
+        targetNode.ownerDocument.documentElement.style.userSelect = "none";
```

Comment:
Not this situation issue as Im 100% that documentElement will never have custom userSelect style,
But styles usually updated, saving previous value/state and restoring it.